core/jvmMain/src/kotlinx/io/buffer/JvmPrimitivesOperations.kt

```diff
@@ -4,112 +4,82 @@ package kotlinx.io.buffer
 
 import kotlinx.io.internal.*
 
-actual inline fun Buffer.loadShortAt(offset: Int): Short {
+public actual inline fun Buffer.loadShortAt(offset: Int): Short {
     return buffer.getShort(offset)
 }
 
-actual inline fun Buffer.loadShortAt(offset: Long): Short {
-    return buffer.getShort(offset.toIntOrFail { "offset" })
+public actual inline fun Buffer.loadShortAt(offset: Long): Short {
+    return buffer.getShort(offset.toIntOrFail("offset"))
 }
 
-actual inline fun Buffer.loadIntAt(offset: Int): Int {
+public actual inline fun Buffer.loadIntAt(offset: Int): Int {
     return buffer.getInt(offset)
 }
 
-actual inline fun Buffer.loadIntAt(offset: Long): Int {
-    return buffer.getInt(offset.toIntOrFail { "offset" })
+public actual inline fun Buffer.loadIntAt(offset: Long): Int {
+    return buffer.getInt(offset.toIntOrFail("offset"))
 }
 
-actual inline fun Buffer.loadLongAt(offset: Int): Long {
+public actual inline fun Buffer.loadLongAt(offset: Int): Long {
     return buffer.getLong(offset)
 }
 
-actual inline fun Buffer.loadLongAt(offset: Long): Long {
-    return buffer.getLong(offset.toInt()) // TODO: toIntOrFail
+public actual inline fun Buffer.loadLongAt(offset: Long): Long {
+    return buffer.getLong(offset.toIntOrFail("offset"))
 }
 
-actual inline fun Buffer.loadFloatAt(offset: Int): Float {
+public actual inline fun Buffer.loadFloatAt(offset: Int): Float {
     return buffer.getFloat(offset)
 }
 
-actual inline fun Buffer.loadFloatAt(offset: Long): Float {
-    return buffer.getFloat(offset.run { toIntOrFail("offset") })
+public actual inline fun Buffer.loadFloatAt(offset: Long): Float {
+    return buffer.getFloat(offset.toIntOrFail("offset"))
 }
 
-actual inline fun Buffer.loadDoubleAt(offset: Int): Double {
+public actual inline fun Buffer.loadDoubleAt(offset: Int): Double {
     return buffer.getDouble(offset)
 }
 
-actual inline fun Buffer.loadDoubleAt(offset: Long): Double {
-    return buffer.getDouble(offset.toIntOrFail { "offset" })
+public actual inline fun Buffer.loadDoubleAt(offset: Long): Double {
+    return buffer.getDouble(offset.toIntOrFail("offset"))
 }
 
-/**
- * Write regular signed 32bit integer in the network byte order (Big Endian)
- */
-actual inline fun Buffer.storeIntAt(offset: Int, value: Int) {
+public actual inline fun Buffer.storeIntAt(offset: Int, value: Int) {
```

Comment:
Why `KDoc` is removed?

Reply:
Currently, KDoc is left only on expect declarations, to simplify things.
Copy-pasting it on actual declaration will be our last action